src/main/java/teammates/ui/webapi/EnrollStudentsAction.java

```diff
@@ -60,14 +60,16 @@ JsonResult execute() {
                     .build());
         });
 
+        List<StudentAttributes> existingStudents = logic.getStudentsForCourse(courseId);
```

Comment:
I wanted to decouple these two so that I could reuse the students pulled (therefore reducing DB read). This is probably the only reason this PR exists.